templates/_components/link.html

```diff
@@ -8,4 +8,4 @@
   {% if new_tab %}
     target="_blank" rel="noopener noreferrer"
   {% endif %}
->{{ text }}</a>
+>{{ text }}</a>{{ append_after }}
```

Comment:
Why do we need to handle this in the component rather than in the containing element?

Reply:
Annoyingly, if you have EditorConfig set to add a new line at the end of your file, Slippers + HTML will add an extra space after the anchor element, meaning you can't line up a full stop (for example) after a link. This was the easiest way I could find to fix this, rather than wrapping things in `spaceless` tags all over the templates.

Open to other ideas though!

Reply:
Oof, that's going to catch out various editor configs, this seems like a solid solution 👍